target/linux/qoriq/generic/target.mk

```diff
@@ -1,3 +1,5 @@
 define Target/Description
-        Build firmware images for NXP QorIQ boards in the AMP series.
+        Build firmware images for all NXP QorIQ boards in the 
```

Comment:
I would remove the all here as it's misleading, we currently only support M300?

Reply:
hi stintel you are correct. Thank you. I will change